File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/controllers/gains/PIDGains.java
package org.firstinspires.ftc.teamcode.control.controllers.gains;

public class PIDGains {

    public final double kP, kI, kD;

    public PIDGains(
            double kP,
            double kI,
            double kD
    ) {
        this.kP = kP;
        this.kI = kI;
        this.kD = kD;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/controllers/coefficients/PIDGains.kt
package org.firstinspires.ftc.teamcode.control.controllers.coefficients

data class PIDGains @JvmOverloads constructor(
    val kP: Double,
    val kI: Double,
    val kD: Double,
    val maxOutputWithIntegral: Double = Double.POSITIVE_INFINITY
)
